.github/workflows/dotnet-build.yml

```diff
@@ -138,10 +150,46 @@ jobs:
       - name: Display dotnet info
         run: dotnet --info
 
-      - name: Build CommunityToolkit.Maui
-        run: dotnet build -c Release ${{ env.PathToLibrarySolution }}
+      - name: 'Build CommunityToolkit.Maui.Analyzers'
+        run: dotnet build ${{ env.PathToCommunityToolkitAnalyzersCsproj }} -c Release
+
+      - name: 'Build CommunityToolkit.Maui.Analyzers.CodeFixes'
+        run: dotnet build ${{ env.PathToCommunityToolkitAnalyzersCodeFixCsproj }} -c Release
+
+      - name: 'Build CommunityToolkit.Maui.Camera.Analyzers'
+        run: dotnet build ${{ env.PathToCommunityToolkitCameraAnalyzersCsproj }} -c Release
+
+      - name: 'Build CommunityToolkit.Maui.Camera.Analyzers.CodeFixes'
+        run: dotnet build ${{ env.PathToCommunityToolkitCameraAnalyzersCodeFixCsproj }} -c Release
+
+      - name: 'Build CommunityToolkit.Maui.MediaElement.Analyzers'
+        run: dotnet build ${{ env.PathToCommunityToolkitMediaElementAnalyzersCsproj }} -c Release
+
+      - name: 'Build CommunityToolkit.Maui.MediaElement.Analyzers.CodeFixes'
+        run: dotnet build ${{ env.PathToCommunityToolkitMediaElementAnalyzersCodeFixCsproj }} -c Release
+
+      - name: 'Build CommunityToolkit.Maui.SourceGenerators'
+        run: dotnet build ${{ env.PathToCommunityToolkitSourceGeneratorsCsproj }} -c Release
+
+      - name: 'Build CommunityToolkit.SourceGenerators.Internal'
+        run: dotnet build ${{ env.PathToCommunityToolkitSourceGeneratorsInternalCsproj }} -c Release
+
+      - name: 'Build CommunityToolkit.Maui.Camera'
+        run: dotnet build ${{ env.PathToCommunityToolkitCameraCsproj }} -c Release -p:PackageVersion=${{ env.NugetPackageVersionCamera }} -p:Version=${{ env.NugetPackageVersion }}
+
+      - name: 'Build CommunityToolkit.Maui.MediaElement'
+        run: dotnet build ${{ env.PathToCommunityToolkitMediaElementCsproj }} -c Release -p:PackageVersion=${{ env.NugetPackageVersionMediaElement }} -p:Version=${{ env.NugetPackageVersion }}
+
+      - name: 'Build CommunityToolkit.Maui.Maps'
+        run: dotnet build ${{ env.PathToCommunityToolkitMapsCsproj }} -c Release -p:PackageVersion=${{ env.NugetPackageVersionMaps }} -p:Version=${{ env.NugetPackageVersion }}
 
-      - name: Run CommunityToolkit.Maui.UnitTests
+      - name: 'Build CommunityToolkit.Maui.Core'
+        run: dotnet build ${{ env.PathToCommunityToolkitCoreCsproj }} -c Release -p:PackageVersion=${{ env.NugetPackageVersion }} -p:Version=${{ env.NugetPackageVersion }}
+
+      - name: 'Build CommunityToolkit.Maui'
+        run: dotnet build ${{ env.PathToLibrarySolution }} -c Release -p:PackageVersion=${{ env.NugetPackageVersion }} -p:Version=${{ env.NugetPackageVersion }} 
```

Comment:
this should be a library, not solution

Reply:
I used the Library SLN on purpose to help ensure the pipeline always builds all of the projects. 

In the future, a PR may add a new library but forget to update the pipeline. This ensures that the pipeline will still compile the new code until we remember to add it as a separate step.

Reply:
Sorry, in that case I still don't see the benefit of building each project individually to build the whole solution later.
I have no strong objections merging the PR, but I still voting keeping it as simple as possible (less code). Once we get the issue with build investigation we can back to the changes.